praw/helpers.py

```diff
@@ -111,6 +112,88 @@ def valid_redditors(redditors, sub):
             if resp['ok']]
 
 
+def all_submissions(reddit_session, subreddit, highest_timestamp=None, lowest_timestamp=0, verbosity=1):
+    """Yield submissions between ``highest_timestamp`` and ``lowest_timestamp``
```

Comment:
Neat idea, I like it! It will save a dozen of requests and make the code cleaner.